sync-core/src/test/java/com/cloudant/common/CouchTestBase.java

```diff
@@ -57,7 +57,7 @@
             System.getProperty("test.couch.ignore.compaction",Boolean.FALSE.toString()));
 
     public static final Boolean IGNORE_AUTH_HEADERS = Boolean.valueOf(
-            System.getProperty("test.couch.ignore.auth.headers",Boolean.FALSE.toString()));
+            System.getProperty("test.couch.ignore.auth.headers",Boolean.TRUE.toString()));
```

Comment:
The previous authentication header tests didn't rely on the server being set up with the correct credentials. For cookie auth for us to be able to test if we correctly use the `_session` API we need to have valid credentials for couchdb to use.